app/controllers/locations_controller.rb

```diff
@@ -42,6 +42,46 @@ def update_ips
     end
   end
 
+  def bulk_upload
+    @upload_form = UploadForm.new
+  end
+
+  def upload_locations_csv
+    @upload_form = UploadForm.new(upload_params)
+    if @upload_form.invalid?
+      render :bulk_upload
+    else
+      @parent_organisation = current_organisation
+      BulkUpload::BulkUpload.add_location_data(@upload_form.data, @parent_organisation)
+      @parent_organisation.validate
+      render("upload_summary")
```

Comment:
could we rename this view "upload_locations_csv" so that we don't need to explicitly render a view that is not the same name as the controller method?